python/ngraph/utils/broadcasting.py

```diff
@@ -34,15 +34,15 @@ def get_broadcast_axes(left_shape, right_shape, axis):
     For example, an output shape of `{2,5,6,2,8}` and input shape of `{2,6}` means
     that the broadcast axes must be `{1,3,4}`.
     """
-    axes_indexes = list(range(0, len(left_shape)))
+    axes_indexes = list(range(0, len(output_shape)))
     if axis is None:
-        right_begin = len(left_shape) - len(right_shape)
```

Comment:
If we're changing names from left, right to in, out let's be consistent and rename all variables.